python/lsst/meas/deblender/baseline.py

```diff
@@ -675,23 +824,65 @@ def _fitPsf(fp, fmask, pk, pkF, pkres, fbb, peaks, peaksF, log, psf, psffwhm,
             img, varimg, psfChisqCut1, psfChisqCut2, psfChisqCut2b,
             tinyFootprintSize=2,
             ):
-    """!
+    """
     Fit a PSF + smooth background model (linear) to a small region around the peak.
-
-    @param[in]     fp            Footprint
-    @param[in]     fmask         the Mask plane for pixels in the Footprint
-    @param[in]     pk            the Peak within the Footprint that we are going to fit a PSF model for
-    @param[in]     pkF           the floating-point coordinates of this peak
-    @param[in,out] pkres         a PerPeak object that will hold the results for this peak
-    @param[in]     fbb           the bounding-box of this Footprint (a Box2I)
-    @param[in]     peaks         a list of all the Peak objects within this Footprint
-    @param[in]     peaksF        a python list of the floating-point (x,y) peak locations
-    @param[in,out] log           pex Log object
-    @param[in]     psf           afw PSF object
-    @param[in]     psffwhm       FWHM of the PSF, in pixels
-    @param[in]     img           the Image in which this Footprint finds itself
-    @param[in]     varimg        Variance plane
-    @param[in]     psfChisqCut*  floats: cuts in chisq-per-pixel at which to accept the PSF model
+    This routine uses a linear least squares algorithm to fit the peak (and neighboring peaks)
+    to the PSF.
+    To make the comparison more robust the peak is fit with and without including
```

Comment:
"...is fit *both* with and without..."? Seems clearer to me, but I'm just being picky.